includes/admin/tools/data/class-give-tools-delete-test-donors.php

```diff
@@ -311,9 +311,22 @@ public function get_data() {
 				$this->update_option( $this->step_on_key, '0' );
```

Comment:
@raftaar1191 When you work any file, Please check other stuff also like documentation,  deprecated or refactor function, code formatting or any logic improvement suggestion.

For Ref: https://github.com/WordImpress/Give/commit/17fa71a64ea32ef7c086b033084002ad49be6d7a